### ISSUE TITLE
A11y issues on 'Create a Book' page

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#4218]
There are three accessibility issues on the 'Create a new book' page:
1. The title element does not identify the contents or purpose of this page (example: https://axe.deque.com/issues/30ead635-549f-4201-8bdb-8757177d8cd5)
2. There are no indicators of which fields are required before user submission (see #4219)
3. If there is an error in user input, the form fields with errors are not identified in text, text alternative, or via programmatic association. The tool tip is not announced in any way. (example: https://axe.deque.com/issues/dad1b448-3415-47c6-ae4a-c8fb18fcf8ec)

These issues were reported by Indiana University

**Comment by SteelWagstaff** [comment]
Rather than fixing on our current page, I recommend making sure we address if/when we implement https://github.com/pressbooks/pressbooks/issues/2856

**Comment by cmurtagh** [comment]
Yeah, I think I like that option too. I'll submit it for next sprint, and fall back to this if we can't do it for some reason.

**Comment by cmurtagh** [comment]
Closing in favour of https://github.com/pressbooks/pressbooks/issues/2856
